### ISSUE TITLE
[Slider]: aria-valuenow attribute is missing on the slider's scale

### ISSUE THREAD

**Issue by niyap** [UI5/webcomponents#13829]
### Bug Description

Each element with role="slider" should have aria-valuemin, aria-valuemax and aria-valuenow attributes presented. Currently aria-valuenow attribute is missing on the element with role="slider" which causes accessibility issue reported by AXE tool.

### Affected Component

ui5-slider

### Expected Behaviour

aria-valuenow attribute should be presented on the element with role="slider"

### Steps to Reproduce

1. Open a sample that has ui5-slider presented
2. Use AXE tool to analyse the page
3. Check the reported issues
...

### Priority

High

### UI5 Web Components Version

latest

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
